mod.js

```diff
@@ -0,0 +1,91 @@
+/* @ts-self-types="./mod.d.ts" */
+import { sendRequestCore } from "./src/commands/send-request.js";
+import logger from "./src/logger.js";
+
+export async function jsonr(filePathOrUrl, options = {}) {
+  // Convert SDK options to internal args format
+  const args = {
```

Comment:
Let's rename args so it's properties are full names instead of single letters. So h becomes headers, etc.